adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -275,7 +275,17 @@ private AuthenticationResult tryAcquireTokenSilent(final AuthenticationRequest a
 
         if (shouldTrySilentFlow(authenticationRequest)) {
             Logger.v(TAG, "Try to acquire token silently, return valid AT or use RT in the cache.");
-            authenticationResult = acquireTokenSilentFlow(authenticationRequest);
+            try {
```

Comment:
Agree on the commit. I was also thinking to add the request parameter into getResultFromBrokerResponse. But I was worried to change the function parameters. It's cool that you already have the fix.